src/libraries/Common/tests/System/Security/Cryptography/X509Certificates/RevocationResponder.cs

```diff
@@ -375,14 +381,14 @@ private static void Trace(string trace)
                 Console.WriteLine(trace);
             }
         }
+    }
 
-        internal enum DelayedActionsFlag : byte
-        {
-            None = 0,
-            Ocsp = 0b1,
-            Crl = 0b10,
-            Aia = 0b100,
-            All = 0b11111111
-        }
+    public enum DelayedActionsFlag : byte
```

Comment:
This needed to be changed to public for test method parameters; enclosing type is internal so move it out.